robyn/router.py

```diff
@@ -117,6 +118,9 @@ def __init__(self) -> None:
         self.global_middlewares: List[GlobalMiddleware] = []
         self.route_middlewares: List[RouteMiddleware] = []
```

Comment:
we should also add a field of `self.authentication_handler: Optional[AuthenticationHandler] = None`